### PR TITLE
Update Auth to Android N

### PR DESCRIPTION
**Pull request by danielpassos**
See: [AGDROID-581](https://issues.jboss.org/browse/AGDROID-581)

### CODE REVIEW

**Comment by danielpassos** [comment]
@corinnekrych @secondsun 👀 ?

File: AuthExamples/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <uses-permission android:name="android.permission.INTERNET"/>
 
     <application
-        android:allowBackup="true"
+        android:allowBackup="false"
         android:icon="@drawable/ic_launcher"
```
Comment (jcesarmobile) [inline AuthExamples/app/src/main/AndroidManifest.xml]: should be `@mipmap/ic_launcher` instead of `@drawable/ic_launcher`

**Comment by jcesarmobile** [comment]
I'm not able to make basic auth work, I always get 401 error even after login.
It works if I login with digest, don't logout and go to basic. (using bacon-corinnekrych.rhcloud.com)

File: AuthExamples/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (danielpassos) [inline AuthExamples/app/src/main/AndroidManifest.xml]: I have changed it in `debug` file instead of the `src`. Good catch!

**Comment by corinnekrych** [comment]
👁

**Comment by corinnekrych** [comment]
Same issue as @jcesarmobile here is the flow that fails for me:
* go to HttpBasic authentication
* click retrieve data => 401 as expected
* click login => sucessgull logged but shortly after 401 to retrieve the list
If I close the app and then start again, go to Basic auth and login directly the flow works.

**Comment by corinnekrych** [comment]
@danielpassos here a PR on you PR to remove login/logout buttons:
https://github.com/danielpassos/aerogear-android-cookbook/pull/6

still having issue though when doing digest http request and then basic http afterward, I run into the error: java.lang.IllegalStateException: Login Called on service which was already logged in.

Any idea?

**Comment by corinnekrych** [comment]
@danielpassos @jcesarmobile https://github.com/danielpassos/aerogear-android-cookbook/pull/7
to remove error: java.lang.IllegalStateException: Login Called on service which was already logged in.

**Comment by jcesarmobile** [comment]
LGTM

**Pull request merged**